src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ApiController.cs

```diff
@@ -95,12 +95,12 @@ public async Task<IActionResult> Create(CreateApiViewModel model)
             shellSettings["Secret"] = Guid.NewGuid().ToString();
             shellSettings["RecipeName"] = model.RecipeName;
 
-            if (!IsDefaultShell() && string.IsNullOrWhiteSpace(shellSettings.RequestUrlHost) && string.IsNullOrWhiteSpace(shellSettings.RequestUrlPrefix))
+            if (String.IsNullOrWhiteSpace(shellSettings.RequestUrlHost) && String.IsNullOrWhiteSpace(shellSettings.RequestUrlPrefix))
```

Comment:
I removed `IsDefaultShell()` here because it is already checked before line 69

            if (!IsDefaultShell())
            {
                return Forbid();
            }